client/state/index.js

```diff
@@ -47,5 +47,8 @@ export function createReduxStore() {
 	) {
 		createStoreWithMiddleware = compose( createStoreWithMiddleware, window.devToolsExtension() );
 	}
-	return createStoreWithMiddleware( createStore )( reducer );
-};
+	if ( initialState === null ) {
+		initialState = {};
+	}
```

Comment:
Hmmm…we seem to have added a default value, which I believe will be assigned if `initialState` were otherwise `null`, so I'm not sure this block will _ever_ be executed.